blog/_posts/2016-09-30-Thank-you-all-from-melbourne.md

```diff
@@ -0,0 +1,50 @@
+---
+title: The end of an amazing Summer(winter) Of Code!!
+layout: post
+created_at: Sun sept 30 2016
+permalink: blog/2016-09-30-amazing end of the winter
+current: blog
+author: ramya
+categories:
+- student-posts
+---
+
+The Rails Girls Summer of Code is one of  the best things that happened to me. After finishing my course at the university I got an opportunity to take part in the program representing the Melbourne team.
+I was very happy and excited by the fact that I will be contributing to an Open source project for 3 months under the guidance of all the coaches and [Katrina Owen](https://twitter.com/kytrinyx) as a mentor. And yes, it was kicked off on 1st of July!
+
+I worked on the project [exercism](http://exercism.io/) and contributed to both x-api and its web client and following summarizes the skills that I have gained over the past three months.
+
+**Ruby**: Prior to this, I had worked on other object oriented programming language like Java. The biggest challenge for me was picking up Ruby and writing clean and elegant ruby code. Thanks to all coaches I got improved by the end of the program.
+
+**GIT**: I learnt many useful techniques about GIT like squashing the commits, cherry pick the commits, rebasing and more. It was useful to know how important to write meaningful commit messages, titles and descriptions for PR especially when working on open source projects.
+
+**Test driven development**: I learnt Minitest testing framework when working on different issues. Writing tests first helped me understand what I really want from the code. It ensures changes and new code don’t break the expectations of existing code. I had a good fun writing code to pass tests and then refactor the code and see if it still passes my test.
+
+
+Working with people geek company like Culture Amp I learnt many things. Attending their culture x design was very useful. Few take away points:
+
+ - Stay positive at the work, including with the people that are working with us.
+ - Treating coworkers with respect and do our best to see    issues from
+   their point of view.
+ - Being open to feedback  can give us an awareness of blind spots, help us to learn, and become better at what we do.
+
+It was a great pleasure working with these amazing minds.
+
+<img src="/img/blog/2016/goodbye-from-vegemite.jpg" alt="Team Vegemite" width="600">
```

Comment:
- As explained [here](https://github.com/rails-girls-summer-of-code/summer-of-code/blob/gh-pages/CONTRIBUTING.md#images), each image published needs a line with image credit (i.e. who took the photo or which site the picture is from). Could you add that still?
- please remove `width="600"` in all `img` tags — the width property is already defined in our css, so you don't need to add it in the markdown file as well (and it also means that the image will not be cut on small displays).